scripts/core/editor3/helpers/resizeDraftSelection.js

```diff
@@ -0,0 +1,13 @@
+import {getBlockAndOffset} from '../helpers/highlights';
+
+export function resizeDraftSelection(stretchLeft, stretchRight, selection, editorState, limitedToSingleBlock = false) {
+    const nextLeft = getBlockAndOffset(editorState, selection, -stretchLeft, false, limitedToSingleBlock);
+    const nextRight = getBlockAndOffset(editorState, selection, stretchRight, true, limitedToSingleBlock);
+
+    return selection.merge({
+        anchorKey: nextLeft.block.getKey(),
+        focusKey: nextRight.block.getKey(),
+        anchorOffset: selection.getIsBackward() ? nextRight.offset : nextLeft.newOffset,
```

Comment:
IMO the keys should check for selection.getIsBackward() too.

Reply:
I've added the check, thanks